mcs/class/corlib/Test/System/TypeTest.cs

```diff
@@ -5106,6 +5106,16 @@ public void IsByRefLike_ArrayOfByrefLike_TLE ()
 			typeof(UserByRefLikeStruct).MakeArrayType ();
 		}
 
+		[Test]
+		public void GetInterfacesOnArrayOfGenericArgumentsToArraySpecialInterfaceGtd ()
+		{
+			// Regression test for https://github.com/mono/mono/issues/7095#issuecomment-470465597
+			// The assertion here isn't very important; what matters is that the runtime doesn't crash.
+			Type ilist_arg = typeof(IList<>).GetGenericArguments () [0];
+			Type[] ilist_arg_array = ilist_arg.MakeArrayType ();
```

Comment:
This doesn't seem right - `MakeArrayType` returns a `Type` not an array of types.

Reply:
Oh. Yes, oops.